src/V3SenExprBuilder.h

```diff
@@ -42,9 +42,9 @@ class SenExprBuilder final {
                                                         // has an update statement in m_preUpdates
     std::unordered_set<VNRef<AstNode>> m_hasPostUpdate;  // Likewis for m_postUpdates
 
-    V3UniqueNames m_currNames{"__Vtrigcurr__expression"};  // For generating unique current value
-                                                           // signal names
-    V3UniqueNames m_prevNames{"__Vtrigprev__expression"};  // Likewise for previous values
+    V3UniqueNames m_currNames{"trigcurr__expression"};  // For generating unique current value
+                                                        // signal names
+    V3UniqueNames m_prevNames{"trigprev__expression"};  // Likewise for previous values
```

Comment:
```suggestion
    V3UniqueNames m_currNames{"trigcurrexpr"};  // For generating unique current value
                                                        // signal names
    V3UniqueNames m_prevNames{"trigprevexpr"};  // Likewise for previous values
```

Double underscore indicates an escape so should be avoided.  Seems clearest to separate reset of name if no underscores at all.